[Internal review] Splitting network management tool

cathay4t opened the pull request:
```
Splitting network management tools into:
 * Core     - iproute2, NetworkManager, dhclient, dnsmasq
 * Desktop  - Core + Bluetooth, WIFI, IPSec VPN, Cellular, NM GUI tools
 * Server   - Core + OVS, team, ADSL, NM server config files,
              source routing, cloud setup
 * Devel    - Core + devel subpackages, nmstate
```

cathay4t commented:
__This is not a upstream pull request, but for internal review only__

tyll commented on configs/sst_networking-management-devel.yaml:
```diff
@@ -0,0 +1,24 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Desktop Network Management
```
The name is wrong (seems to be a copy-paste leftover from the desktop file)

cathay4t replied:
Changed to `Network Management in Development`

tyll commented on configs/sst_networking-management-core.yaml:
```diff
@@ -0,0 +1,15 @@
+version: 1
+data:
+  name: Core Network Management
+  description: The Core Network Management tools for the local host. This includes NetworkManager and related tools.
```
It might be nicer to wrap the lines at 88 characters.

tyll commented on configs/sst_networking-management-core.yaml:
```diff
@@ -0,0 +1,15 @@
+  description: The Core Network Management tools for the local host. This includes NetworkManager and related tools.
+  maintainer: sst_networking
+  packages:
+  - iproute2
```
Sorting the packages case-insensitive would be nice.

tyll commented on configs/sst_networking-management-server.yaml:
```diff
@@ -0,0 +1,34 @@
+  - NetworkManager-tui
+
+  # Open vSwitch
+  - NetworkManager-ovs
```
Does nmstate-plugin-ovsdb belong here instead of in the development environment?

cathay4t replied:
Moved nmstate to server group.

tyll commented on configs/sst_networking-management-desktop.yaml:
```diff
@@ -0,0 +1,43 @@
+  - usb_modeswitch-data
+
+  # VPN
+  - NetworkManager-libreswan
```
the libreswan plugin might also be useful in the server use case.

tyll commented on configs/sst_networking-management-desktop.yaml:
```diff
@@ -0,0 +1,43 @@
+  - NetworkManager-tui
+
+  # Modem
+  - ModemManager
```
For IoT use cases, ModemManager might also be needed. Not sure, if we should track this, too, or add this as a IoT section to the server workload.

jwboyer committed:
Merge pull request #584 from notroj/infra-services-jws

Add mod_jk/mod_cluster to httpd workflow.

square1-box committed:
Update buildroot-eln.yaml 2021-05-24-13:00

square1-box committed:
Update buildroot-package-relations json 2021-05-24-13:00

square1-box committed:
Update eln-buildroot-workload 2021-05-24-22:17

square1-box committed:
Update buildroot-eln.yaml 2021-05-24-22:17

square1-box committed:
Update buildroot-eln.yaml 2021-05-24-23:27

square1-box committed:
Update eln-buildroot-workload 2021-05-24-23:27

square1-box committed:
Update eln-buildroot-workload 2021-05-25-00:31

square1-box committed:
Update buildroot-eln.yaml 2021-05-25-00:31

square1-box committed:
Update eln-buildroot-workload 2021-05-25-02:45

square1-box committed:
Update buildroot-eln.yaml 2021-05-25-02:45

mkosek committed:
python-pytest-runner is deprecated and should not be used

jwboyer committed:
Merge pull request #587 from mkosek/unwanted-pytest-runner

python-pytest-runner is deprecated and should not be used

square1-box committed:
Update eln-buildroot-workload 2021-05-25-15:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-25-15:25

square1-box committed:
Update eln-buildroot-workload 2021-05-25-16:22

square1-box committed:
Update buildroot-eln.yaml 2021-05-25-16:22

square1-box committed:
Update buildroot-eln.yaml 2021-05-25-17:35

square1-box committed:
Update eln-buildroot-workload 2021-05-25-17:35

square1-box committed:
Update buildroot-eln.yaml 2021-05-26-01:23

square1-box committed:
Update eln-buildroot-workload 2021-05-26-01:23

square1-box committed:
Update buildroot-eln.yaml 2021-05-26-02:20

square1-box committed:
Update eln-buildroot-workload 2021-05-26-02:20

square1-box committed:
Update buildroot-eln.yaml 2021-05-26-03:05

square1-box committed:
Update eln-buildroot-workload 2021-05-26-03:05

square1-box committed:
Update eln-buildroot-workload 2021-05-26-04:13

square1-box committed:
Update buildroot-eln.yaml 2021-05-26-04:13

square1-box committed:
Update eln-buildroot-workload 2021-05-26-22:27

square1-box committed:
Update buildroot-eln.yaml 2021-05-26-22:27

square1-box committed:
Update eln-buildroot-workload 2021-05-27-00:46

square1-box committed:
Update buildroot-eln.yaml 2021-05-27-00:46

square1-box committed:
Update eln-buildroot-workload 2021-05-27-02:06

square1-box committed:
Update buildroot-eln.yaml 2021-05-27-02:06

square1-box committed:
Update buildroot-package-relations json 2021-05-27-02:06

square1-box committed:
Update eln-buildroot-workload 2021-05-27-22:31

square1-box committed:
Update buildroot-eln.yaml 2021-05-27-22:31

square1-box committed:
Update eln-buildroot-workload 2021-05-27-23:45

square1-box committed:
Update buildroot-eln.yaml 2021-05-27-23:45

square1-box committed:
Update buildroot-package-relations json 2021-05-27-23:45

square1-box committed:
Update eln-buildroot-workload 2021-05-28-01:06

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-01:06

square1-box committed:
Update buildroot-package-relations json 2021-05-28-01:06

square1-box committed:
Update eln-buildroot-workload 2021-05-28-03:24

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-03:24

square1-box committed:
Update eln-buildroot-workload 2021-05-28-04:20

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-04:20

square1-box committed:
Update eln-buildroot-workload 2021-05-28-05:31

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-05:31

square1-box committed:
Update buildroot-package-relations json 2021-05-28-13:10

square1-box committed:
Update eln-buildroot-workload 2021-05-28-20:24

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-20:24

square1-box committed:
Update eln-buildroot-workload 2021-05-28-21:28

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-21:28

square1-box committed:
Update eln-buildroot-workload 2021-05-28-22:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-22:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-28-23:16

square1-box committed:
Update eln-buildroot-workload 2021-05-28-23:16

square1-box committed:
Update eln-buildroot-workload 2021-05-29-00:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-00:25

square1-box committed:
Update eln-buildroot-workload 2021-05-29-01:22

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-01:22

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-02:15

square1-box committed:
Update eln-buildroot-workload 2021-05-29-02:15

square1-box committed:
Update eln-buildroot-workload 2021-05-29-03:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-03:25

square1-box committed:
Update eln-buildroot-workload 2021-05-29-09:28

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-09:28

square1-box committed:
Update eln-buildroot-workload 2021-05-29-11:45

square1-box committed:
Update buildroot-eln.yaml 2021-05-29-11:45

square1-box committed:
Update eln-buildroot-workload 2021-05-30-02:30

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-02:30

square1-box committed:
Update eln-buildroot-workload 2021-05-30-03:26

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-03:26

square1-box committed:
Update eln-buildroot-workload 2021-05-30-04:37

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-04:37

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-07:30

square1-box committed:
Update eln-buildroot-workload 2021-05-30-07:30

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-08:41

square1-box committed:
Update eln-buildroot-workload 2021-05-30-08:41

square1-box committed:
Update eln-buildroot-workload 2021-05-30-13:40

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-13:40

square1-box committed:
Update eln-buildroot-workload 2021-05-30-14:50

square1-box committed:
Update buildroot-eln.yaml 2021-05-30-14:50

square1-box committed:
Update eln-buildroot-workload 2021-05-31-12:25

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-12:25

square1-box committed:
Update eln-buildroot-workload 2021-05-31-13:17

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-13:17

square1-box committed:
Update eln-buildroot-workload 2021-05-31-14:28

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-14:28

square1-box committed:
Update eln-buildroot-workload 2021-05-31-15:34

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-15:34

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-16:45

square1-box committed:
Update eln-buildroot-workload 2021-05-31-16:45

square1-box committed:
Update eln-buildroot-workload 2021-05-31-17:55

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-17:55

square1-box committed:
Update buildroot-package-relations json 2021-05-31-17:55

square1-box committed:
Update eln-buildroot-workload 2021-05-31-18:58

square1-box committed:
Update buildroot-eln.yaml 2021-05-31-18:58

square1-box committed:
Update buildroot-package-relations json 2021-05-31-18:58

square1-box committed:
Update eln-buildroot-workload 2021-06-01-02:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-02:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-03:15

square1-box committed:
Update eln-buildroot-workload 2021-06-01-03:15

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-04:25

square1-box committed:
Update eln-buildroot-workload 2021-06-01-04:25

square1-box committed:
Update eln-buildroot-workload 2021-06-01-05:30

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-05:30

square1-box committed:
Update eln-buildroot-workload 2021-06-01-06:33

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-06:33

asamalik committed:
Merge pull request #583 from uril/spice_8

spice-vdagent is wanted

asamalik committed:
Merge pull request #580 from notroj/move-whois

Move whois between RHEL teams.

square1-box committed:
Update eln-buildroot-workload 2021-06-01-07:46

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-07:46

square1-box committed:
Update eln-buildroot-workload 2021-06-01-11:22

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-11:22

square1-box committed:
Update buildroot-package-relations json 2021-06-01-11:22

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-12:19

square1-box committed:
Update eln-buildroot-workload 2021-06-01-12:19

square1-box committed:
Update buildroot-package-relations json 2021-06-01-12:19

square1-box committed:
Update eln-buildroot-workload 2021-06-01-13:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-13:26

square1-box committed:
Update buildroot-package-relations json 2021-06-01-13:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-14:33

square1-box committed:
Update eln-buildroot-workload 2021-06-01-14:33

square1-box committed:
Update eln-buildroot-workload 2021-06-01-15:47

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-15:47

square1-box committed:
Update eln-buildroot-workload 2021-06-01-21:35

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-21:35

square1-box committed:
Update eln-buildroot-workload 2021-06-01-22:47

square1-box committed:
Update buildroot-eln.yaml 2021-06-01-22:47

kdudka committed:
sst_graphics_infrastructure-codecs: remove vorbis

It was pulling vorbis-tools into the compose despite vorbis-tools
is on the list of unwanted packages of sst_cs_plumbers.

jwboyer committed:
Merge pull request #590 from kdudka/vorbis-tools

sst_graphics_infrastructure-codecs: remove vorbis

square1-box committed:
Update eln-buildroot-workload 2021-06-02-11:35

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-11:35

square1-box committed:
Update eln-buildroot-workload 2021-06-02-12:46

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-12:46

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-13:58

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-17:22

square1-box committed:
Update eln-buildroot-workload 2021-06-02-17:22

square1-box committed:
Update eln-buildroot-workload 2021-06-02-18:28

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-18:28

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-19:21

square1-box committed:
Update eln-buildroot-workload 2021-06-02-19:21

square1-box committed:
Update eln-buildroot-workload 2021-06-02-20:30

square1-box committed:
Update buildroot-eln.yaml 2021-06-02-20:30

square1-box committed:
Update eln-buildroot-workload 2021-06-03-10:15

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-10:15

square1-box committed:
Update eln-buildroot-workload 2021-06-03-11:35

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-11:35

square1-box committed:
Update eln-buildroot-workload 2021-06-03-17:20

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-17:20

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-18:31

square1-box committed:
Update eln-buildroot-workload 2021-06-03-18:31

square1-box committed:
Update eln-buildroot-workload 2021-06-03-19:30

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-19:30

square1-box committed:
Update buildroot-eln.yaml 2021-06-03-20:47

square1-box committed:
Update eln-buildroot-workload 2021-06-03-20:47

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-01:20

square1-box committed:
Update eln-buildroot-workload 2021-06-04-01:20

square1-box committed:
Update eln-buildroot-workload 2021-06-04-02:18

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-02:18

square1-box committed:
Update eln-buildroot-workload 2021-06-04-03:17

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-03:17

square1-box committed:
Update eln-buildroot-workload 2021-06-04-04:30

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-04:30

jridky committed:
Add gphoto2 as unwanted for system management

square1-box committed:
Update eln-buildroot-workload 2021-06-04-10:40

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-10:40

jwboyer committed:
Merge pull request #591 from jridky/master

Add gphoto2 as unwanted for system management

square1-box committed:
Update eln-buildroot-workload 2021-06-04-11:49

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-11:49

asamalik committed:
change placeholders to real packages in c9s virt

asamalik committed:
include arch packages in their arches only

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-19:38

square1-box committed:
Update eln-buildroot-workload 2021-06-04-19:38

square1-box committed:
Update eln-buildroot-workload 2021-06-04-20:50

square1-box committed:
Update buildroot-eln.yaml 2021-06-04-20:50

square1-box committed:
Update eln-buildroot-workload 2021-06-05-02:20

square1-box committed:
Update buildroot-eln.yaml 2021-06-05-02:20

square1-box committed:
Update eln-buildroot-workload 2021-06-05-03:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-05-03:26

square1-box committed:
Update buildroot-eln.yaml 2021-06-05-16:28

square1-box committed:
Update eln-buildroot-workload 2021-06-05-16:28

square1-box committed:
Update eln-buildroot-workload 2021-06-05-18:45

square1-box committed:
Update buildroot-eln.yaml 2021-06-05-18:45

square1-box committed:
Update eln-buildroot-workload 2021-06-06-00:20

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-00:20

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-01:30

square1-box committed:
Update eln-buildroot-workload 2021-06-06-01:30

square1-box committed:
Update eln-buildroot-workload 2021-06-06-02:10

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-02:10

square1-box committed:
Update eln-buildroot-workload 2021-06-06-03:18

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-03:18

square1-box committed:
Update eln-buildroot-workload 2021-06-06-21:45

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-21:45

square1-box committed:
Update buildroot-package-relations json 2021-06-06-21:45

square1-box committed:
Update buildroot-eln.yaml 2021-06-06-22:55

square1-box committed:
Update eln-buildroot-workload 2021-06-07-00:55

square1-box committed:
Update buildroot-eln.yaml 2021-06-07-00:55

square1-box committed:
Update buildroot-package-relations json 2021-06-07-00:55

asamalik committed:
remove the buildroot configs

asamalik committed:
remove the buildroot configs

asamalik committed:
Merge pull request #593 from asamalik/virt

[do not merge yet] Virt fixes

cgwalters committed:
edge: Require nss-altfiles

In practice, rpm-ostree requires this today.  It's not
expressed as a hard dependency because we may do something else
in the future.

hroncok committed:
python3.10 needs to be in ELN

hroncok committed:
We don't want python3.9 in ELN

asamalik committed:
Merge pull request #596 from hroncok/patch-7

python3.10 needs to be in ELN

jwboyer committed:
Merge pull request #594 from cgwalters/nss-altfiles

edge: Require nss-altfiles

ptoscano committed:
rhsm: drop subscription-manager-initial-setup-addon

subscription-manager-initial-setup-addon is deprecated, not working with
the newer Anaconda APIs, and thus being removed from RHEL 9:
https://bugzilla.redhat.com/show_bug.cgi?id=1941904

Newer versions of Anaconda can register systems during the installation,
so this is less of a concern.

rhsm-gtk is removed too, as it is build only in RHEL because
subscription-manager-initial-setup-addon is too.

jwboyer committed:
Merge pull request #597 from ptoscano/rhsm-updates

rhsm: drop subscription-manager-initial-setup-addon

jwboyer committed:
Merge pull request #582 from hhorak/remove-pymongo

Removing pymongo from openstack workload

marceloleitner committed:
sst_networking: explode tools workload

into multiple workloads, so that workload pages have a better, more
focused, list of dependencies for each of these packages.

They are stand-alone packages, and understanding their deps is more
important aggregating them into a single workload.

Signed-off-by: Marcelo Ricardo Leitner <marcelo.leitner@gmail.com>

asamalik committed:
Merge pull request #600 from marceloleitner/explode-tools-workload

sst_networking: explode tools workload

wchwhite committed:
sst_kernel_rats: Add sysfsutils config workload

- sysfsutils needs to be included in RHEL 9.

asamalik committed:
Merge pull request #601 from rh-chwhite/master

sst_kernel_rats: Add sysfsutils config workload

voxik committed:
Drop rubygem-abrt from Ruby package set.

There is not ABRT in ELN therefore there is no use for Ruby ABRT support.

RajeshKumarSrivastava committed:
Added config for automotive packages based on c8s to find dependencies

jwboyer committed:
Merge pull request #602 from voxik/patch-6

Drop rubygem-abrt from Ruby package set.

mizdebsk committed:
Drop c9s-specific override for Maven

Make Maven workloads for eln and c9s identical.

asamalik committed:
Merge pull request #604 from mizdebsk/apps-java-unify

Drop c9s-specific override for Maven

asamalik committed:
Merge pull request #603 from RajeshKumarSrivastava/automotive_c8s_package_manifest_config

Added config for automotive packages based on c8s to find dependencies

RajeshKumarSrivastava committed:
Add an environment with epmty package list for automotive and refer the same in the automotive workload

Commit:
sst_kernel_maintainers: add kernel-debug* packages to develdebug list

We miss the kernel-debug* packages on the list. They are being always
built on eln now, so we should list them too.

Signed-off-by: Herton R. Krzesinski <herton@redhat.com>

jwboyer committed:
Merge pull request #606 from herton/kmaint_missing3

sst_kernel_maintainers: add kernel-debug* packages to develdebug list

asamalik committed:
Merge pull request #605 from RajeshKumarSrivastava/automotive_c8s_package_manifest_configs

Add an environment with empty package list for automotive and refer the same in automotive workload

FaramosCZ committed:
Add new "mariadb-pam" subpackage of "mariadb" package

jwboyer committed:
Merge pull request #607 from FaramosCZ/master

Add new "mariadb-pam" subpackage of "mariadb" package

tiran committed:
sst IdM: Mark custodia as unwanted

IPA 4.9.6 no longer depends on python3-custodia.

See: https://pagure.io/freeipa/issue/8882
Signed-off-by: Christian Heimes <cheimes@redhat.com>

jridky committed:
Add new packages openexr and imath

jridky committed:
Add new package watchdog

jwboyer committed:
Merge pull request #609 from jridky/master

Add new package

snbueno committed:
Create the x3270 workload

jwboyer committed:
Merge pull request #610 from snbueno/patch-1

Create the x3270 workload

asamalik committed:
Merge pull request #608 from tiran/unwanted-custodia

sst IdM: Mark custodia as unwanted

mizdebsk committed:
Add unwanted sst_cs_apps Java packages

jwboyer committed:
Merge pull request #611 from mizdebsk/unwanted-java

Add unwanted sst_cs_apps Java packages

asamalik committed:
exclude cockpit-machines from ppc64le

drsmith2 committed:
Marked babeltrace as unwanted.

drsmith2 committed:
Merge branch 'minimization:master' into master

asamalik committed:
Merge pull request #612 from drsmith2/master

Mark babeltrace as unwanted.

RajeshKumarSrivastava committed:
Update repository-c8s.yaml

enable `aarch64` to get included in the results for automotive-c8s-package-manifest

asamalik committed:
Merge pull request #613 from RajeshKumarSrivastava/patch-1

Update repository-c8s.yaml

RajeshKumarSrivastava committed:
Update packages

asamalik committed:
Merge pull request #614 from RajeshKumarSrivastava/c8s_package_config_updates

Update automotive packages

mkosek committed:
Rename sst_identity_management workload to ssg_idm

Identity Management subsystem got internally split and renamed
to IdM subsystem group (SSG).

mkosek committed:
ctdb is part of selected clustered Samba use cases

More information:
https://access.redhat.com/articles/5090441

jwboyer committed:
Merge pull request #615 from mkosek/add-ctdb

Add ctdb

rhawalsh committed:
SST_PLATFORM_STORAGE Marked python-psutil as unwanted.

jwboyer committed:
Merge pull request #616 from rhawalsh/FAST-content_set

SST_PLATFORM_STORAGE Marked python-psutil as unwanted.

cathay4t committed:
sst_networking: Remove ModemManager-vala from explicit list

`ModemManager-vala` should be in buildroot repo as it was in RHEL8
till customer requested.

Signed-off-by: Gris Ge <fge@redhat.com>
